tfhe-zk-pok/src/curve_api.rs

```diff
@@ -116,6 +116,10 @@ pub trait CurveGroupOps<Zp>:
     fn to_le_bytes(self) -> impl AsRef<[u8]>;
     fn double(self) -> Self;
     fn normalize(self) -> Self::Affine;
+    fn validate_projective(&self) -> bool {
+        Self::validate_affine(&self.normalize())
+    }
+    fn validate_affine(affine: &Self::Affine) -> bool;
```

Comment:
In my mind it would be clearer to have a `trait ValidateAffine { fn validate_affine(&self) -> bool;}` and the contraint `Affine: ValidateAffine` here instead (names could be changed)
Having a trait method which does not use `self` is a bit confusing

Reply:
Makes sense